sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java

```diff
@@ -890,7 +890,7 @@ private InternalObjectNode getInternalObjectNode() {
         InternalObjectNode internalObjectNode = new InternalObjectNode();
         String uuid = UUID.randomUUID().toString();
         internalObjectNode.setId(uuid);
-        BridgeInternal.setProperty(internalObjectNode, "mypk", uuid);
+        BridgeInternal.setProperty(internalObjectNode, "mypk", "wiqkclagxrdqclmvuzcsomihrkxbbikwypqrcgrhpgkylztdxxirzwwkmleovdrikggqupfcclpaowyetbywvbeyegniacruvzncxflfzrwnhtdubzrezefqhtyznagotfxtcynnyderhryvbtpoxbuyfkkwsmoydfmglwzgqobraysfidipfsybbgsromwwiuygkuzbitkwzvzuenfjvxkklcjomrasmddllqfiqmgmcbmnmikpvyzbdbuitfjuohwbqfyueqjvbwehgwhosooehtglncoyiundhhqrsbkazhuxjzqoteouwexvzchkhuukpkqkdlkfhhqbgcvrhzizljrkfujebedsfdrvzdvavxrwpkyqvnvbbqvoylvxejetqvzqaakrsvbnilfngjgdavkdwfgxqvhjatrccvsbjpueenjljgouzoqdweckcjjhfiloefucsmloniyklseuztfhkzufbzntnvlaziaqqoayilyxssmfafvawxxnhyyaosnlsujsemzwvpirnbkjtfbggaamzcocdvsebdkxkyfvkakvangpgmrqqctlrbblrmmwchqliypeuuqtbiozgtfjtwjdhhowetaltnlpwqokckxpbfrpyshcygzrxdzhnffjagppnaeyrzuhrofoyurrythnpefgtiybfmckzwvsgntsudnqfqcudeporevbhgvlkaavdypfiahusbzwcgsigspfpxlqbxhqfzbrrcrgtlcbokvrtrghobknoxcaelnhchwaekpqepimanaqagajdpzbsaoepnkkfrxsavjkpxeawhrhacnnutphuzngxnpzrtujcfwvmhxuskdmmoyjdtrmqvbgehwpfseiyjsmugjpcbndigkufynwnuolasltqxfirxvkqcibculpqbarejhtufquoqzvvpwfszzpsjovmajrfviuozdscztuhavlszbdncrxdsmoikkmxwenfkqoomdzxbvw");
```

Comment:
what is this change?

Reply:
I was doing some test with a large payload to check the time spent on decode stage. 

has reverted it back.